### PR TITLE
Add objects and roles

### PR DESCRIPTION
**Pull request by mattreim**

### CODE REVIEW

File: main.js
Comment (Jey-Cee) [inline main.js:1]: scheduleron is correct, look at https://github.com/dresden-elektronik/deconz-rest-plugin/blob/b126d6be93b1d90ec644f6474dfc67efa45276da/thermostat.cpp#L14

Comment (mattreim) [reply main.js:1]: I didn't find it in the [documentation ](https://dresden-elektronik.github.io/deconz-rest-doc/)but I can add it additionally.

**Comment by mattreim** [comment]
This is what my Bosch thermostat looks like with the changes.

Raumthermostat

File: main.js
Comment (Jey-Cee) [reply main.js:1]: I know, the documentation does not show all of the possible datapoints 😔

Comment (mattreim) [reply main.js:1]: Unfortunately

**Pull request merged**